client/src/App.vue

```diff
@@ -1,10 +1,16 @@
 <script setup lang="ts">
 import { RouterView } from 'vue-router'
 import BluecrewFooter from '@/components/BluecrewFooter.vue'
+import NavBarWrapper from '@/components/NavBarWrapper.vue'
 </script>
 
 <template>
-  <RouterView />
+  <v-app>
+    <NavBarWrapper />
+    <v-main>
```

Comment:
We need to make sure that the pages display below the navbar. This depends on what you do elsewhere, but I think one way of achieving this is to add the ``padding-top`` property to the ``v-main`` element (the amount of padding should be the same as the height of the navbar.